### PR TITLE
Magician router

### PR DESCRIPTION
**Pull request by mntedev**
* I have completed the following tasks.**
1. added home, calculator, and quote components.
2. added header component.
3. added styling for header
4. added styling for home, calculator, and quote component.

**=> home component screenshot.**

image

**=> calculator component screenshot.**

image
**=> quote component screenshot.**
image

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
 import React, { useState } from 'react';
 import calculate from './logic/calculate';
 import AnswerScreen from './AnswerScreen';
-
+import Header from './Header';
+/* eslint-disable*/
 const Calculator = () => {
```
Comment (DammyShittu) [inline src/components/Calculator.js:6]: - [ ] Good job implementing the SPA as outlined. However, the general layout of the Calculator component is still required to match the wireframe. Kindly reduce the size of your calculator to improve the look, as it covers almost the entire screen.

  Your design | Provided design
  :---:|:---:
  router-1 | router-2

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @worashf,

✔️ SPA functionality

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mntedev** [comment]
I have changed the header and calculator components stsyling.
image

**Review by arslanbisharat — APPROVED** [review]
Hi @worashf.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**